SignalServiceKit/src/Contacts/ContactsUpdater.m

```diff
@@ -88,11 +59,11 @@ - (void)lookupIdentifiers:(NSArray<NSString *> *)identifiers
 
     [self contactIntersectionWithSet:[NSSet setWithArray:identifiers]
                              success:^(NSSet<SignalRecipient *> *recipients) {
-                                 if (recipients.count > 0) {
-                                     success(recipients.allObjects);
-                                 } else {
-                                     failure(OWSErrorMakeNoSuchSignalRecipientError());
+                                 if (recipients.count == 0) {
+                                     DDLogDebug(
+                                         @"%@ in %s no contacts are Signal users", self.logTag, __PRETTY_FUNCTION__);
```

Comment:
I decided not to make this change because I was afraid of breaking something and didn't want to take the time to audit everywhere this was used to ensure it was safe.  Thanks for doing the legwork.